### ISSUE TITLE
Mobile support?

### ISSUE THREAD

**Issue by Void48** [catppuccin/github#58]

**Comment by justTOBBI** [comment]
I dont think this will be possible unless you make the use of some custom client with theme support

**Comment by andreasgrafen** [comment]
I would assume that the whole mobile app UI is build on a completely different basis. Based on this assumption I'd say that all of the CSS is pretty much useless for mobile anyway. :/
